tests/main-missing-owner.test.js

```diff
@@ -0,0 +1,13 @@
+// @ts-check
+
+process.env.GITHUB_REPOSITORY = "actions/create-github-app-token";
+delete process.env.GITHUB_REPOSITORY_OWNER;
+
+// Verify `main` exits with an error when `GITHUB_REPOSITORY_OWNER` is missing.
+(async () => {
+  try {
+    await import("../main.js");
+  } catch (error) {
+    console.error(error.message);
+  }
+})();
```

Comment:
The async wrapper shouldn't be necessary, Node 18+ supports top-level await

Reply:
If I remove the async IIFE wrapper, TypeScript warns[^1] about using top-level `await` in a file that’s not a module:
> 'await' expressions are only allowed at the top level of a file when that file is a module, but this file has no imports or exports. Consider adding an empty 'export {}' to make this file a module.ts(1375)

There are other ways to make that error go away:

- I can add `export {}` to the bottom of the file.
- I can change the file extension to `.mjs`.
- I can remove `// @ts-check`.

I’m happy to change this—do you prefer one of those options over the wrapper?

[^1]:  Because of `// @ts-check`.

Reply:
ah yes I would remove `// @ts-check` in test files in general, we don't need to test types, we test code

Reply:
Done!

- I removed `// @ts-check` from test files, in https://github.com/actions/create-github-app-token/pull/56/commits/02f90f51d8ae7b1f3490df19c54ac4676e9af4af.
- I removed the async IIFE wrappers, in https://github.com/actions/create-github-app-token/pull/56/commits/fabe5404a563d7c338e6019fb78642619983b66d.